### ISSUE TITLE
Overview attribute fields have differing tone color for selection (when marking tickets via bulk actions or on an individual basis)

### ISSUE THREAD

**Issue by Suhopoljac** [zammad/zammad#5253]
### Used Zammad Version

6.3.1

### Environment

- Installation method: any
- Operating system (if you're unsure: `cat /etc/os-release` ): any
- Database + version:  any
- Elasticsearch version: any
- Browser + version:  Ms Edge v.126.0.2592.87 (official build) (64-bit)

### Actual behaviour

When selecting tickets in the overview panel, the attributes (down, in the middle of the screen) display one color tone for groups and another for other attributes:

Group
<img width="188" alt="image" src="https://github.com/zammad/zammad/assets/101350124/a07c28f8-dd49-4f59-a653-9800bb45533f">

State
<img width="166" alt="image" src="https://github.com/zammad/zammad/assets/101350124/890add1a-c49c-4046-a9f6-d4980762cfb2">

Priority:
<img width="185" alt="image" src="https://github.com/zammad/zammad/assets/101350124/e8514467-169c-42bd-bc6f-8639b4f1c6e7">

### Expected behaviour

When selecting tickets in the overview panel, the attributes (down, in the middle of the screen) should display the same color tone for groups and every other attribute:

### Steps to reproduce the behaviour

1) Select at least one ticket in the overviews. 
2) Click on the group and see that the selected option is displayed in a different color tone than the other attributes. 

### Support Ticket

Ticket#10157646

### I'm sure this is a bug and no feature request or a general question.

yes

**Comment by rolfschmidt** [comment]
I think this is because of the differences between the select and the tree_select, I believe this will get changed in the new stack. Not sure, if we should address this in the old stack.

**Comment by dvuckovic** [comment]
Yes, agreed: the field design across different select/treeselect/autocomplete flavors was unified in the new stack, and they look much more closer to each other. For the legacy app this makes not a lot of sense at this point, due to the mix of native/custom fields.

I'll close this issue as already fixed in the new stack.
